uaclient/util.py

```diff
@@ -652,6 +643,22 @@ def is_config_value_true(config: "Dict[str, Any]", path_to_value: str):
         )
 
 
+def get_value_from_dict(
+    config: "Dict[str, Any]", path_to_value: str, default: "Any" = None
+):
+    """Return the value present at a given path_to_value"""
+    value = config
+    default_value = {}  # type: Any
+    paths = path_to_value.split(".")
+    leaf_value = paths[-1]
+    for key in paths:
+        if key == leaf_value:
+            default_value = default
+
+        value = value.get(key, default_value)
```

Comment:
This seems to be the same code we are using on `util.is_config_value_true`. I think we can refactor to it to avoid this duplication

Reply:
I agree it was duplication, which is why I pulled out the common `util.get_value_from_dict` from `util.is_config_value_true`. is_config_value_true adds a bit more "flexibility" allowing people to provide either True, TRUE, true or TruE etc in environment values.   I realize that yaml.safe_load supports conversion of YES,True TRUE etc to python's `True`, but our environment variable handling doesn't. I've filed #1411 to track that separately